.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
```

Comment:
Maybe we want to recheck these. This and the below mobile device details would seem more applicable for a frontend application! Same with the reproduce steps above. What do you think?

Reply:
> Maybe we want to recheck these. This and the below mobile device details would seem more applicable for a frontend application! Same with the reproduce steps above. What do you think?

Agreed. Please check new ones.

Reply:
Solid. Extra nice to have the other templates too!

Reply:
We could think of creating a guide to adding architectures.

Reply:
> We could think of creating a guide to adding architectures.

yeah, we need a bit more clarity into the process for other architectures but possible.